### ISSUE TITLE
JRuby installation not getting detected

### ISSUE THREAD

**Issue by SwathiMystery** [LinkedInAttic/white-elephant#2]
My system has JRuby installed, but the while running "ant", which runs setup.sh, to check for jruby, does not detect the installed JRuby.

Snippet :  
swathi@localhost:~/white-elephant/server$ jruby -v
jruby 1.7.3 (1.9.3p385) 2013-02-21 dac429b on Java HotSpot(TM) 64-Bit Server VM 1.6.0_35-b10 [linux-amd64]

While running ant :
swathi@localhost:~/white-elephant/server$ ant
Buildfile: /home/swathi/white-elephant/server/build.xml

setup:
     [exec] Making sure RVM is not installed
     [exec] [v] done
     [exec] Checking for rbenv
     [exec] [v] done
     [exec] Checking for JRuby
     [exec] JRuby 1.7.3 is not installed
     [exec] Installing JRuby 1.7.3
     [exec] [v] Created temporary directory
     [exec] [v] Found wget

Also, I see that it gets stuck at wget. Anything that I'm missing here ?

Thanks.
- Swathi

**Comment by matthayes** [comment]
Sorry for the confusion, I'll update the logging to make it more clear.  The setup installs JRuby into a local directory at server/.rbenv independent of whatever JRuby version is installed on your machine already.  This is to avoid issues with different versions of JRuby and different setups.

How long does it hang on wget?  When you see this message it is downloading the JRuby binary.  I should add a message indicating this.  Can you try letting it complete?

**Comment by SwathiMystery** [comment]
Ah! No problem. Idea of clear logging is good!
I saw the setup.sh and realized that it is independent. Sorry about that. It makes sense. 
However, if the version expected is already installed, would it be a good idea to skip?

wget got hung like for a longer time. So, I had to quit. I however cross-checked the URL of s3, it is right!
Is it possible to add a progress message in function installJRuby() for wget?

Thanks much.
